content/pt-br/docs/concepts/services-networking/network-policies.md

```diff
@@ -0,0 +1,345 @@
+origens no caso de tráfego de entrada ou destino no tráfego de saída.
+
+__namespaceSelector__ *e* __podSelector__: Uma entrada `to`/`from` única que permite especificar 
+ambos `namespaceSelector` e `podSelector` e seleciona um conjunto de Pods dentro de um namespace.
+Seja cuidadoso em utilizar a sintaxe YAML correta; essa política:
+
+```yaml
+  ...
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          user: alice
+      podSelector:
+        matchLabels:
+          role: client
+  ...
+```
+contém um único elemento `from` permitindo conexões de Pods com a label `role=client` em 
+namespaces com a _label_  `user=alice`. Mas *essa* política:
+
+```yaml
+  ...
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          user: alice
+    - podSelector:
+        matchLabels:
+          role: client
+  ...
+```
+
+contém dois elementos no conjunto `from` e permite conexões de Pods no namespace local com 
+a _label_ `role=client`, *OU* de qualquer outro Pod em qualquer outro namespace que tenha 
+a label `user=alice`.
+
+Quando estiver em dúvida, utilize o comando `kubectl describe` para verificar como o 
+Kubernetes interpretou a política.
+
+__ipBlock__: Isso seleciona um conjunto particular de faixas de IP a serem permitidos como 
+origens no caso de entrada ou destinos no caso de saída. Devem ser considerados IPs externos
+ao cluster, uma vez que os IPs dos Pods são efêmeros e imprevisíveis.
+
+Os mecanismos de entrada e saída do cluster geralmente requerem que os IPs de origem ou destino
+sejam reescritos. Em casos em que isso aconteça, não é definido se deve acontecer antes ou 
+depois do processamento da `NetworkPolicy` que corresponde a esse tráfego, e o comportamento
+pode ser diferente para cada plugin de rede, provedor de nuvem, implementação de `Service`, etc.
+
+No caso de tráfego de entrada, isso significa que em alguns casos você pode filtrar os pacotes 
+de entrada baseado no IP de origem atual, enquanto que em outros casos o IP de origem que 
+a `NetworkPolicy` atua pode ser o IP de um `LoadBalancer` ou do Nó em que o Pod está executando.
+
+No caso de tráfego de saída, isso significa que conexões de Pods para `Services` que são reescritos
+para IPs externos ao cluster podem ou não estar sujeitos a políticas baseadas no campo `ipBlock`.
+
+## Políticas padrão
+
+Por padrão, se nenhuma política existir no namespace, então todo o tráfego de entrada e saída é 
+permitido de e para os pods nesse namespace. Os exemplos a seguir permitem a você mudar o 
+comportamento padrão nesse namespace.
+
+### Bloqueio padrão de todo tráfego de entrada
+
+Você pode criar uma política padrão de isolamento para um namespace criando um objeto `NetworkPolicy` 
+que seleciona todos os pods mas não permite o tráfego de entrada para esses pods.
+
+{{< codenew file="service/networking/network-policy-default-deny-ingress.yaml" >}}
```

Comment:
Hey @rikatz this file doesn't exist in pt-br example files directory, that's why the build is broken.
Error log:
```
3:42:36 PM: Error: Error building site: "/opt/build/repo/content/pt-br/docs/concepts/services-networking/network-policies.md:223:1": failed to render shortcode "codenew": failed to process shortcode: "/opt/build/repo/layouts/shortcodes/codenew.html:14:16": execute of template failed: template: shortcodes/codenew.html:14:16: executing "shortcodes/codenew.html" at <readFile $filename>: error calling readFile: stat /opt/build/repo/content/pt-br/examples/service/networking/network-policy-default-deny-ingress.yaml: no such file or directory
```

Reply:
@rikatz can you fix this problem?
Thanks